src/Stratis.Bitcoin.IntegrationTests/API/ApiSteps.cs

```diff
@@ -193,15 +192,15 @@ private void a_real_transaction()
 
         private void the_block_with_the_transaction_is_mined()
         {
-            this.block = this.firstStratisPowApiNode.GenerateStratisWithMiner(1).Single();
-            this.firstStratisPowApiNode.GenerateStratisWithMiner(1);
+            this.block = TestHelper.MineBlocks(this.firstStratisPowApiNode, 1).BlockHashes[0];
```

Comment:
Can these 2 lines be replaced by
`TestHelper.MineBlocks(this.firstStratisPowApiNode, 2).BlockHashes[0];`?

Reply:
Ah yeah, well spotted!